scripts/sync_scheduled_agents.py

```
@@ -0,0 +1,151 @@
+import argparse
+import json
+import subprocess
+import sys
+import uuid
+from pathlib import Path
+
+AGENTS_DIR = Path(__file__).parent.parent / "infra" / "scheduled_agents"
+ENVIRONMENT_ID = "env_01ELefm6C9bVcvqCVgbptR5N"
+
+
+def run_claude_api(action: str, trigger_id: str = None, body: dict = None) -> dict:
```

Ruff (0.15.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)